File: 0062-unique-paths/0062-unique-paths.cpp
class Solution {
public:
    int uniquePaths(int m, int n) {
        m=m-1;
        n=n-1;
        int r=min(m,n);
        long long ans=1;
        for(int i=0;i<r;i++){
            ans=ans*(n+m-i)/(i+1);
        }
        return ans;
    }
};

File: 0062-unique-paths/0062-unique-paths.py
class Solution:
    def uniquePaths(self, m: int, n: int) -> int:
        m = m - 1
        n = n - 1
        mini = min(m, n)
        ans = 1
        for i in range(mini):
            ans = ans * (n + m - i) // (i + 1)
        return int(ans)